pa4/checker-pa4.ml

```diff
@@ -768,26 +778,36 @@ let main () = begin
 	(* Consolidated List of User and Base classes *)
     (* This adds the bases classes to our class inheritence mapping so that they aren't treated as
 -	undeclared classes. *)
-        let base_classes = ["Bool"; "IO"; "Int"; "String"; "Object"] in
-        let user_classes = List.map (fun ((_,cname),_,_) -> cname) ast in
+    let base_classes = ["Bool"; "IO"; "Int"; "String"; "Object"] in
+    let user_classes = List.map (fun ((_,cname),_,_) -> cname) ast in
 	let all_classes = base_classes @ user_classes in
 	let all_classes = List.sort compare all_classes in
-        let vertices = ref [] in
+    let vertices = ref [] in
```

Comment:
I know that this isn't directly related to this pull request, but can we rename this variable? Calling it "vertices" is relevant to what it's used for in the topological sort, but doesn't really make sense outside of that context.